integration-tests/pom.xml

```diff
@@ -122,6 +122,17 @@
   </dependencies>
   <build>
     <plugins>
+      <plugin>
+        <artifactId>maven-jar-plugin</artifactId>
```

Comment:
This won't work very well because artifacts for this module are not meant to be deployed to Maven Central, see [here]( https://github.com/datastax/java-driver/blob/2997d775a091cfee614e49d20c98f5b29518d7c0/integration-tests/pom.xml#L318-L323).

As far as i can tell, you only need one class in this module: `CassandraTestResource`. I'm afraid you are going to have to copy-paste it.

It turns out that we already have 2 copies of this class:

1. deployment/src/test/java/com/datastax/oss/quarkus/CassandraTestBase.java
2. integration-tests/src/test/java/com/datastax/oss/quarkus/CassandraTestResource.java

So I suggest that you create a third copy for now. But we definitely should look into creating a `test-infra` module to host test utilities like this one.

Reply:
It does work, see:
https://jenkins-drivers.build.dsinternal.org/job/tools/job/cassandra-quarkus/job/java2791/3/testReport/com.datastax.oss.quarkus/
The FruitIT is executed succesfully

Reply:
But it won't if someone copies the quickstart and uses it as a standalone project. The test-jar is coming from the build itself, not from Maven Central. So the quickstart cannot be a standalone project anymore, if you keep this dependency.

Reply:
ah. you are right. Done